root/main/400.tt

```diff
@@ -4,15 +4,7 @@
 
         <p><strong>[%- l("Sorry, there was a problem with your request.") -%]</strong></p>
 
-        <p>
-            <strong>[%- l('Error message: ') -%]</strong>
-            [% IF message %]
-                <code>[%- message | html -%]</code>
-            [% ELSE %]
-                <code>[%- l('(No details about this error are available)') -%]</code>
-            [% END %]
-        </p>
-
+        [% INCLUDE 'main/info/error.tt' %]
```

Comment:
When there is no stack trace in the stash, `error.tt` resolves exactly to the above.